.cirrus.yml

```diff
@@ -22,14 +22,14 @@ env:
     UBUNTU_PROJECT: "ubuntu-os-cloud"
 
     # Images exist on GCP already
-    CENTOS_9_IMAGE_NAME: "centos-stream-9-v20221102"
-    CENTOS_8_IMAGE_NAME: "centos-stream-8-v20221206"
-    DEBIAN_IMAGE_NAME: "debian-11-bullseye-v20221206"
-    FEDORA_IMAGE_NAME: "fedora-cloud-base-gcp-37-beta-1-5-x86-64"
-    FEDORA_PRIOR_IMAGE_NAME: "fedora-cloud-base-gcp-36-20220506-n-0-x86-64"
-    UBUNTU_IMAGE_NAME: "ubuntu-2204-jammy-v20230114"
-    UBUNTU_PRIOR_IMAGE_NAME: "ubuntu-2004-focal-v20230113"
-    UBUNTU_PRIOR2_IMAGE_NAME: "ubuntu-1804-bionic-v20230112"
+    CENTOS_9_IMAGE_NAME: "centos-stream-9-v20230306"
+    CENTOS_8_IMAGE_NAME: "centos-stream-8-v20230306"
+    DEBIAN_IMAGE_NAME: "debian-11-bullseye-v20230306"
+    FEDORA_IMAGE_NAME: "fedora-cloud-base-gcp-38-beta-1-3-x86-64"
+    FEDORA_PRIOR_IMAGE_NAME: "fedora-cloud-base-gcp-37-1-7-x86-64"
+    UBUNTU_IMAGE_NAME: "ubuntu-2204-jammy-v20230302"
+    UBUNTU_PRIOR_IMAGE_NAME: "ubuntu-2004-focal-v20230302"
+    UBUNTU_PRIOR2_IMAGE_NAME: "ubuntu-1804-bionic-v20230324"
     UBUNTU_SNAP_IMAGE_NAME: "ubuntu-2204-jammy-v20230114"
```

Comment:
You probably want to update this image to the same as the latest one, `ubuntu-2204-jammy-v20230302`

Reply:
Thanks, will do. I wasn't sure if there were special considerations for the snap builder image.